WSL/about.md

```diff
@@ -13,19 +13,19 @@ ms.assetid: 3cefe0db-7616-4848-a2b6-9296746a178b
 
 # Windows Subsystem for Linux Documentation
 
-The Windows Subsystem for Linux lets developers run Linux environments -- including most command-line tools, utilities, and applications -- directly on Windows, unmodified, without the overhead of a virtual machine.  
+The Windows Subsystem for Linux lets developers run GNU environment -- including most command-line tools, utilities, and applications -- directly on Windows, unmodified, without the overhead of a virtual machine.  
 
 You can:
 
 1. Choose your favorite Linux distributions [from the Windows Store](https://aka.ms/wslstore).
-1. Run common command-line utilities such as `grep`, `sed`, `awk`, etc. 
-1. Run Bash shell scripts and Linux command-line applications including:  
+1. Run common command-line free software such as `grep`, `sed`, `awk`, etc. 
```

Comment:
Would like to keep as "utilities" since that is a more commonly used qualifier in popular Linux documentation

Reply:
Maybe "Run common command-line free software utilities, such as" ?